src/lib/services/AuthenticationService/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface SendPinBody {
```

Comment:
Is the location of this types file (and its naming) appropriate. I've opted to colocate it with its usage and give in a generic `types.ts` name.

Reply:
by the looks of it, these interfaces are used only in AuthenticationService I guess,

I wouldnt even mind to have it in `AuthenticationService.ts` itself for now. If in case it needs to be used elsewhere then it can be moved to separate file

Reply:
+1